### DISCUSSION TITLE
NEW SECURLY EXPLOIT- Google Translate

### DISCUSSION THREAD

**Discussion by DankFireR** [3kh0/ext-remover#836] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
My information comes from https://ading.dev/blog/posts/securly_bypass.html
By adding #translate.google.com to the end of a blocked website, it'll bypass the block
Be careful though I'm not sure how far it'll work
Oh yeah and it only works for Securly

**Comment by Aka-but-nice** [comment]
It won't work for Securly version `2.98.53`

**Reply by DankFireR** [reply]
rip
